pages/common/install.md

```diff
@@ -0,0 +1,24 @@
+# install
+
+> Copy files and set attributes.
+> Provide a standardized way of manipulating a file/directory's ownership and permissions while copying file or creating directory.
```

Comment:
I was thinking more along the lines of the [kill command](https://github.com/tldr-pages/tldr/blob/master/pages/common/kill.md), which starts by describing what the command actually does ("Sends a signal to a process"), but follows that up by mentioning the use that gives the command its name ("usually related to stopping the process."). In this case, we could mention that a major use of this is to setup programs downloaded or built from source (e.g. copy files to a system location like /usr/local/bin, give them the appropriate permissions/ownership so they can be used by all users, and mark them as executable) hence "install". Does that sound reasonable to you?